packages/edit-site/src/components/page-pages/index.js

```diff
@@ -79,7 +79,7 @@ export default function PagePages() {
 	} = useEntityRecords( 'postType', 'page', queryArgs );
 
 	const { records: authors } = useEntityRecords( 'root', 'user', {
-		who: 'authors',
+		has_published_posts: true,
```

Comment:
I substituted `who` by `has_published_posts` because I'm no longer sure what `who` is about, see [code](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/rest-api/endpoints/class-wp-rest-users-controller.php#L1584C32-L1584C32).

Reply:
That's interesting indeed. I'm wondering if @TimothyBJacobs can shed some light about the nuances with `who`.
```suggestion
		has_published_posts: [ 'page' ],
```
We need to limit to pages though.

Reply:
Oh, [interesting](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/rest-api/endpoints/class-wp-rest-users-controller.php#L1592). So it admits both a boolean and an array.

Reply:
Yeah `who` is weird and not really recommended AFAIK.

Looking at the UI now, it seems like `has_published_posts` makes the most sense. But it does seem like this UI might want to display `draft` pages at some point? In which case, restricting to published posts would be incorrect.

Reply:
Thanks for the feedback, Timothy. I'm addressing it at https://github.com/WordPress/gutenberg/pull/55723